openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/interfaces/sell-item.interface.ts

```diff
@@ -32,4 +32,5 @@ export interface ISellItem extends IItem {
     isOrderItem: boolean;
     isTender: boolean;
     svgImage: boolean;
+    orderIcon:string;
```

Comment:
nit: can you add a space between the name and type here.

```suggestion
    orderIcon: string;
```

Reply:
Fixed